pkg/helm/helm.go

```diff
@@ -115,7 +115,7 @@ func renderChart(namespace, baseValues, overlayValues string, chrt *chart.Chart)
 
 	var sb strings.Builder
 	for _, f := range files {
-		_, err := sb.WriteString(f)
+		_, err := sb.WriteString(f + YAMLSeparator)
```

Comment:
is it ok if we get 2 separators?

Reply:
Have checked the default behavior of `helm template`, it will add additional YAMLSeparator between two resources, no matter whether the original template contains YAMLSeparator or not.

Reply:
@howardjohn if K8s receives an empty blob in a manifest (triggerered by 2 seperators) it ignores it.  Its not tidy, and something that could be optimized, but not really worth the trouble at this time.

Cheers
-steve

Reply:
it's not just about what kubernetes expects, we need to make sure all usages of this output handle it ok as well. I don't know where this is actually used, just wanted to point it out since we had issues in the test framework with extra separators

Reply:
interesting re test infra.

Reply:
yeah if I remember right it would try to do `kubectl apply nil` or something. Nothing that can't be fixed just need to he careful